### ISSUE TITLE
Exception thrown when attempting to activate plugin on windows environment

### ISSUE THREAD

**Issue by dmalshin** [xwp/wp-customize-snapshots#105]
Can't activate the plugin due to a fatal error.
Tested on two different websites, on WP 4.7 and on the previous WP version.
![2bf6bdc744](https://cloud.githubusercontent.com/assets/10954709/20975379/c95efcdc-bcb8-11e6-8bb4-d93f0505c673.png)

**Comment by westonruter** [comment]
Note: The plugin is not yet compatible with WordPress 4.7.

That being said, this error should not be happening on 4.7 or any previous version.

**Comment by westonruter** [comment]
The (attempted) exception being thrown is:

> Plugin dir is not inside of the `wp-content` directory

See https://github.com/xwp/wp-customize-snapshots/blob/629033dd3704945aa78e72986441425925c1b1a2/php/class-plugin-base.php#L152

It seems that the plugin is having a hard time locating itself on your Windows filesystem.

**Comment by awps** [comment]
As well. I can't activate it.
Looke like this plugin does not have support for symlinks. 
Also, is not a good method to extend a class like this:
```php
class Exception extends \Exception {}
```
IMHO, something like this this is better:
```php
class DoException extends \Exception {}
```
I would love to see this fixed. Should I do a contribution?

![screen-0001](https://user-images.githubusercontent.com/1436534/27263568-58bc22d0-5474-11e7-995c-e1e2ac13f30f.png)

**Comment by westonruter** [comment]
@Smartik89 the name of the exception is fine and is as intended, since it is being namespaced. Please do open a PR with a fix for the symlink issue.

**Comment by Alexander-Shukaev** [comment]
I have the same issue on Linux:

```
[15-Jul-2017 13:34:30 UTC] PHP Fatal error:  Uncaught Error: Class 'CustomizeSnapshots\Exception' not found in /.../wp-content/plugins/customize-snapshots/php/class-plugin-base.php:152
Stack trace:
#0 /.../wp-content/plugins/customize-snapshots/php/class-plugin-base.php(71): CustomizeSnapshots\Plugin_Base->locate_plugin()
#1 /.../wp-content/plugins/customize-snapshots/php/class-plugin.php(56): CustomizeSnapshots\Plugin_Base->__construct()
#2 /.../wp-content/plugins/customize-snapshots/instance.php(15): CustomizeSnapshots\Plugin->__construct()
#3 /.../wp-content/plugins/customize-snapshots/customize-snapshots.php(33): require_once('...')
#4 /.../wp-admin/includes/plugin.php(1882): include('...')
#5 /.../wp-admin/plugins.php(164): plugin_sandbox_scrape('customize-snaps...')
#6 {main}
  thrown in /.../plugins/customize-snapshots/php/class-plugin-base.php on line 152
```

Not a PHP expert at all but a quick search suggests [this](https://stackoverflow.com/a/10001405/1743860).

**Comment by westonruter** [comment]
Please test fix in #139.

**Comment by Alexander-Shukaev** [comment]
You can go ahead with the merge, confirmed.  Thank you.
